packages/dataviews/src/components/dataviews-bulk-actions/index.tsx

```diff
@@ -65,7 +65,7 @@ export function useHasAPossibleBulkAction< Item >(
 	item: Item
 ) {
 	return useMemo( () => {
-		return actions.some( ( action ) => {
+		return actions?.some( ( action ) => {
```

Comment:
This is not related to this PR.

Reply:
Does it make sense to move it out to another PR?